packages/gatsby-source-contentful/src/__tests__/__snapshots__/gatsby-node.js.snap

```diff
@@ -2,17 +2,17 @@
 
 exports[`gatsby-node should add a new blogpost and update linkedNodes 1`] = `
 Object {
-  "author___NODE": "uzfinxahlog0___c15jwOBqpxqSAOy2eOO4S0m",
-  "body___NODE": "uzfinxahlog0___c2XTZRrfbMRcfdyFSo4mrORbodyTextNode",
+  "author___NODE": "uzfinxahlog0___c15jwOBqpxqSAOy2eOO4S0m___Entry",
```

Comment:
Wasn't this changed recently as not to leak this kind of information? That seems like a regression...? Or perhaps this is something unrelated. Just wanted to make sure.

Reply:
As in the type was removed from the ID?

Reply:
If I'm not mistaken this will now leak the node type into production. I'm not entirely sure but I think the node type name may be something like "super_secret_launch_feature", in which case that would now leak to generated pages, right?

Reply:
I believe this is the contentful type, such as "Asset", "Link", "Entry" - not the actual name of the content type.

If necessary I can probably obfuscate it similar to the rest

Reply:
If this really is just the contentful name then I guess it's fine. Wanted to make sure we weren't going to potentially accidentally leak something.